solidstart-tanstackquery-tailwind-modules/src/types/issues-type.ts

```diff
@@ -82,7 +82,7 @@ export type Variables = {
     createdBy?: string;
     mentioned?: string;
     milestone?: string;
-    labels?: string[];
+    labels?: (string | undefined)[];
```

Comment:
Why this? (I couldn't seem to find a reason)

Reply:
This is the only way to solve the type error in this line
![image](https://user-images.githubusercontent.com/49324987/222452643-ecc3a0b7-69fa-491e-bf57-248dd53b6df8.png)

Reply:
There seems to be a mismatch of whether more than one label can be used in `filterBy` I guess.

The type doesn't make sense to me, and array of possible`undefined`s but this can go in a follow up